work_in_progress/model_variance.ipynb

```diff
@@ -0,0 +1,691 @@
+    "\n",
+    "If we want to look at anomalies of future projections rather than absolute, first we need to calculate a historical baseline for our metric of interest. From this baseline and the absolute values provided by future projections, we can calculate anomalies. Use the select tool, or bypass and hardcode the selections for historical 1985-2015. \n",
+    "\n",
+    "### 1a) Select historical data\n",
+    "First we need to select the relevant subset of historical data. In this example, we want to select the statistically downscaled area average over Alameda County of monthly maximum air temperature over our historical baseline period of 1985-2015. We can either use the selection tool (commented out) to choose this subset of historical data; or we can just hard-coded these selections. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc50c6a7-d2a4-4e32-88ef-84f417d86d0f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "selections = ck.Select()\n",
+    "\n",
+    "selections.area_average='Yes'\n",
+    "selections.area_subset='CA counties'\n",
+    "selections.cached_area=['Alameda County']\n",
+    "selections.data_type='Gridded'\n",
+    "selections.downscaling_method=['Statistical']\n",
+    "selections.resolution='3 km'\n",
+    "selections.scenario_historical=['Historical Climate']\n",
+    "selections.time_slice=(1985, 2015)\n",
+    "selections.timescale='monthly'\n",
+    "selections.units='K'\n",
+    "selections.variable='Maximum air temperature at 2m'\n",
+    "\n",
+    "# selections.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "936c41f3-2a0f-4b27-acbd-dfdb106f3930",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### 1b) Retrieve and load data\n",
+    "Next we retrieve the historical data that we have selected, and load it into memory. The historical data may take a minute to load."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cde5f5fb-e69d-4da0-a53b-9d2a3f4d6d54",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hist_data = selections.retrieve()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a080f21c-eb7a-4499-8795-806b4797833b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hist_data = ck.load(hist_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ad54f5e-eba9-4799-98ec-a1f06a5a413d",
+   "metadata": {},
+   "source": [
+    "## Step 2: Calculate weighted temporal mean of historical data\n",
+    "\n",
+    "From monthly timeseries, calculate annual averages (weighted by the number of days in each month), and then average across years."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fa85b245-a1f8-4603-ac51-472445e34292",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from climakitae.explore.uncertainty import weighted_temporal_mean\n",
```

Comment:
Could remove the commented code and move the `weighted_temporal_mean` function to `utils` now that multiple files reference it.

Reply:
It's already in utils, just forgot to remove this commented code out, good catch!